sentinel-core/src/main/java/com/alibaba/csp/sentinel/util/function/Endpoint.java

```diff
@@ -0,0 +1,53 @@
+package com.alibaba.csp.sentinel.util.function;
+
+import java.net.InetSocketAddress;
+
+/**
+ * @author Leo Li
+ */
+public class Endpoint {
+    private Protocol protocol;
+
+    private String host;
+
+    private int port;
+
+    public Endpoint(Protocol protocol, String host, int port) {
+        this.protocol = protocol;
+        this.host = host;
+        this.port = port;
+    }
+
+    public Protocol getProtocol() {
+        return protocol;
+    }
+
+    public void setProtocol(Protocol protocol) {
+        this.protocol = protocol;
+    }
+
+    public String getHost() {
+        return host;
+    }
+
+    public void setHost(String host) {
+        this.host = host;
+    }
+
+    public int getPort() {
+        return port;
+    }
+
+    public void setPort(int port) {
+        this.port = port;
+    }
+
+    public InetSocketAddress getInetSocketAddress() {
```

Comment:
It's strange to put such a piece of code here but I won't consist on it. It looks like no harm.